### PR TITLE
Update m365-feature-descriptions.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about Technical support for partners

fixes #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10301

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/m365-feature-descriptions.md [hunk 1]
```diff
@@ -179,6 +179,8 @@ Office 365 for enterprise follows a role-based access control (RBAC) model: perm
 
 **Partners:** You can select a Microsoft partner and delegate administrative functions, including creating service request tickets. For more information, see the [Partners](/office365/servicedescriptions/office-365-platform-service-description/partners) service description and [Add, change, or delete a subscription advisor partner](/office365/admin/misc/add-partner).
 
+**Technical support for partners:** If you're a partner and need information about specific partner support plans, see the following table. For more information, see [compare partner support plans](https://partner.microsoft.com/support/partnersupport). Partners can also escalate tickets to Microsoft. For more information, See [Escalating to Microsoft] (/partner-center/customer-support#escalating-to-microsoft)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/m365-feature-descriptions.md]:
```suggestion
**Technical support for partners:** If you're a partner and need information about specific partner support plans, see the following table. For more information, see [Compare partner support plans](https://partner.microsoft.com/support/partnersupport). Partners can also escalate tickets to Microsoft. For more information, see [Escalating to Microsoft] (/partner-center/customer-support#escalating-to-microsoft).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, you are referencing a table, but you did not include the table. Could you please include it? Many thanks.

**Comment by msbemba** [comment]
@JohanFreelancer9 I have updated it now. Please copy review

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, could you please include the table provided by the issue raiser? If not, the reference to the table needs to be removed. Thanks.

**Comment by msbemba** [comment]
the table is available on the link and i have removed the reference to the table.

File: microsoft-365/admin/m365-feature-descriptions.md [hunk 2]
```diff
@@ -179,6 +179,8 @@ Office 365 for enterprise follows a role-based access control (RBAC) model: perm
 
 **Partners:** You can select a Microsoft partner and delegate administrative functions, including creating service request tickets. For more information, see the [Partners](/office365/servicedescriptions/office-365-platform-service-description/partners) service description and [Add, change, or delete a subscription advisor partner](/office365/admin/misc/add-partner).
 
+**Technical support for partners:** If you're a partner and need information about specific partner support plans, see [Compare partner support plans](https://partner.microsoft.com/support/partnersupport). Partners can also escalate tickets to Microsoft. For more information, see [Escalating to Microsoft] (/partner-center/customer-support#escalating-to-microsoft)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/m365-feature-descriptions.md]:
```suggestion
**Technical support for partners:** If you're a partner and need information about specific partner support plans, see [Compare partner support plans](https://partner.microsoft.com/support/partnersupport). Partners can also escalate tickets to Microsoft. For more information, see [Escalating to Microsoft](/partner-center/customer-support#escalating-to-microsoft).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, thank you. Just one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**